code/API_definitions/carrier_billing.yaml

```diff
@@ -87,9 +132,46 @@ paths:
               $ref: "#/components/schemas/CreatePayment"
         required: true
       callbacks:
-        eventNotifications:
-          "{$request.body#/webhook.notificationUrl}/eventNotifications":
-            $ref: "#/paths/~1eventNotifications"
+        notifications:
+          "{$request.body#/webhook/notificationUrl}":
+            post:
+              security:
+                - {}
+                - notificationsBearerAuth: []
+              tags:
+                - Payment Notifications
+              summary: Carrier Billing payment notifications
+              operationId: paymentNotification
+              description: |
+                Important: This endpoint is exposed by the API client, accepting requests in the defined format.
+                The Carrier Billing server will call this endpoint whenever any carrier billing related event occurs.
+              parameters:
+                - $ref: "#/components/parameters/x-correlator"
+              requestBody:
+                description: Creates a new carrier billing payment notification
+                content:
+                  application/json:
```

Comment:
Should be: `application/cloudevents+json:`

Reply:
Rationale about this value is this? @bigludo7 : https://github.com/camaraproject/Commonalities/blob/2420cf352633eb1076ee5383963fec2e4387605c/artifacts/notification-as-cloud-event.yaml#L79

Reply:
Yes and here also: https://github.com/camaraproject/Commonalities/pull/86

Reply:
Thanks for clarification!